Encapsulation Exercises/Fare_Calculator/src/PublicVehicleFares/Main.java

```diff
@@ -0,0 +1,58 @@
+package PublicVehicleFares;
+
+import java.util.Scanner;
+public class Main {
+    public static void main (String[] args){
+        Scanner sc = new Scanner(System.in);
+            int option = 0;
+
+            System.out.println(" How many kilometers have you traveled from point A to B? : ");
```

Comment:
simplify your interface. just ask for the distance traveled by the passenger. then use the PublicVehicle class to calculate the fare. then output the total fare.  put necessary input validations